examples/dart/pubspec.yaml

```diff
@@ -8,4 +8,4 @@ environment:
 dependencies:
   dio: ^3.0.0
   cookie_jar: ^1.0.0
```

Comment:
are we making sure cookies persist?

Reply:
Hard to say, never used this lib and it was there earlier. But there is only one class `CookieManager` which is an interceptor and it is added to http client.

Reply:
I think I can add a test for this in the mock server, it will also allow us to enforce that all SDKs know how to handle this.